cfme/tests/containers/test_provider_configuration_menu.py

```diff
@@ -11,6 +11,34 @@
 ]
 
 
+def get_buttons(view):
+    # ensure resume option is exsit
+    resume_option = filter(lambda item: "resume" in item.lower(), view.toolbar.configuration.items)
```

Comment:
*Required*

I would prefer to see list comprehension here instead of a lambda, since its not necessary.

The following achieves the same thing
`assert any(['resume' in item.lower() for item in view.toolbar.configuration.items]), 'Resume not in menu' `

Reply:
@mshriver  I am using the option value in the next steps, so I would not use the ```any``` function. I would change this implementation to use list comprehension